src/main/java/org/dependencytrack/tasks/VulnerabilityAnalysisTask.java

```diff
@@ -149,9 +149,9 @@ private void dispatchComponents(final UUID scanToken, final List<ComponentProjec
 
     private List<ComponentProjection> fetchNextComponentsPage(final PersistenceManager pm, final Project project, final Long lastId) throws Exception {
         try (final Query<Component> query = pm.newQuery(Component.class)) {
-            var filter = "project.active == :projectActive";
+            var filter = "project.inactiveSince == :projectInactiveSince";
             var params = new HashMap<String, Object>();
-            params.put("projectActive", true);
+            params.put("projectInactiveSince", null);
```

Comment:
`null` should be passed as a literal, see https://github.com/DependencyTrack/dependency-track/issues/2540

```suggestion
            var filter = "project.inactiveSince == null";
            var params = new HashMap<String, Object>();
```